broker_app.py

```diff
@@ -178,8 +183,52 @@ def get_spreadsheet(job_id: str):
         )
 
 
+# http://0.0.0.0:5000/schemas?high_level_entity=type&domain_entity=biomaterial&concrete_entity=donor_organism&latest&json
+# http://0.0.0.0:5000/schemas?url=${schemaUrl}&json&deref
+@app.route('/schemas', methods=['GET'])
```

Comment:
Why are we implementing this here and not in core where we have the schemas endpoint?

Reply:
yeah i'll probably remove the retrieving of latest schema here 🤔  i actually want to remove the schema endpoints from core, and have a separate schema service. Or i can just put the dereferencing in Core. Let me think about this a bit more...

Reply:
Unfortunately, given the time remaining for the sprint, i won't be able implement this in Core or separate schema service. I will keep take note of this for now. And track this in zenhub for now.

Reply:
I added this issue for now ebi-ait/hca-ebi-dev-team#345